streaming/rrdpush.c

```diff
@@ -28,6 +28,7 @@
 #define STREAMING_PROTOCOL_VERSION "1.1"
```

Comment:
What's the relationship of this with `STREAMING_PROTOCOL_CURRENT_VERSION`?

Reply:
I think this is the "in between" protocol version that simply sets the NETDATA_PROTOCOL_VERSION and its out there now -- it is ignored if a `ver` is present during the stream establishment

Reply:
The original idea was to have conditions to use versions like the kernel version, but after a talk with Andrew we moved it to an integer version, so as @stelfrag said this is between stream versions.